interview-questions.md

```diff
@@ -14,44 +14,44 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: git is a version control software. Github is a collaboration platform.
 
-Researched answer:
+Researched answer: git is a version control that tracks changes. While Github is a platform where those changes can be uploaded.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: the OR operator
 
-Researched answer:
+Researched answer: The OR operator will return true if one or more values are true.
```

Comment:
What about other operators? Look through relational, equality, and logical operators.